shipment_management/public/js/custom_script.js

```diff
@@ -62,12 +63,12 @@ function create_dialog(frm) {
                 },
                 freeze: 1,
                 callback: function (r) {
-                    sales_orders = []
-                    for (var i = 0; i < frm.doc.items.length; i++) {
+                    let sales_orders = [];
+                    for (let i = 0; i < frm.doc.items.length; i++) {
                         sales_orders.push(frm.doc.items[i].against_sales_order)
                     }
                     sales_orders = Array.from(new Set(sales_orders));
-                    for (var i = 0; i < sales_orders.length; i++) {
+                    for (let i = 0; i < sales_orders.length; i++) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'sales_orders' is not in camel case. (camelcase)](https://app.codacy.com/manual/mail_67/shipment_management/pullRequest?prid=5453114)